draft-ietf-rats-corim.md

```diff
@@ -2508,6 +2524,26 @@ If no entry is found, the comparison MUST return false.
 Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}} after equivalence has been found according to {{sec-comid-integrity-registers}}.
 Note that it is not required for all the entries in the candidate entry to be used during matching: the condition ECT could consist of a subset of the device's register space. In TPM parlance, a TPM "quote" may report all PCRs in Evidence, while a condition ECT could describe a subset of PCRs.
 
+##### Comparison for linear-privlevel entries
+
+The ACS entry value stored under `measurement-values-map` codepoint 15 is a linear privilege level, which must have type `linear-privlevel-type-choice`.
+
+If the entry `linear-privlevel-type-choice` is an `int`, name it PRIV and compare as follows.
+
+*  If the condition ECT value for `measurement-values-map` codepoint 15 is an `int` then an equality comparison is performed on the `int` components.
+The comparison MUST return true if the value of PRIV is equal to the `int` value in the condition ECT.
+
+*  If the condition ECT value for `measurement-values-map` codepoint 15 is an `int-range` tagged with #6.564 then a range inclusion comparison is performed.
+The comparison MUST return true if the value of PRIV is greater than or equal to the `min` value in the condition ECT AND the value of PRIV is less than or equal to than the `max` value in the condition ECT
+
+If the entry `linear-privilege-type-choice` is an `int-range` or `int-range` tagged with #6.564, then comparison with the pair of `opt-int` values MINPRIV and MAXPRIV is as follows.
+
+*  If the condition ECT value for `measurement-values-map` codepoint 15 is an `int` then the comparison MUST return true if and only if MINPRIV and MAXPRIV are equal and finite.
```

Comment:
What is MINPRIV and MAXPRIV here, please set the context, like you did for PRIV Above.

Just say they are constant with Value X or Value Y.

Reply:
If I reread the text above using MAXINT/MININT the wording seems confusing in that max infinity can't be equal to min infinity and infinity can be equal to a finite number at the same time. Or at least I have to be a theoretical mathematition to make it make sense. 

I think the text is trying to say that if the input value is type 'int' then the comparison must exactly match a value that is also of type 'int'. It is noted that MAXINT and MININT are special case values of type 'int' because we don't want to encode large numbers.

Reply:
I've changed the wording. Please take a look.